.npmignore

```diff
@@ -54,3 +54,8 @@ rust-toolchain
 .rustfmt.toml
 .gitattributes
 .travis.yml
+.husky/
+.prettierrc
+crates/
```

Comment:
this one's actually bit unsure - https://unpkg.com/browse/@swc/core@1.2.117/crates/ are these needed in runtime? looks like it's readme / or jsons.